packages/orbit-design-tokens/src/js/transparentColor.ts

```diff
@@ -1,11 +1,16 @@
 import { parseToRgba, rgba, guard } from "color2k";
 
+const isHex = (color: string): boolean => color.startsWith("#");
+const isRgb = (color: string): boolean => color.startsWith("rgb");
+
 const transparentColor = (color: string, opacity: number): string => {
   // tailwind package is using css variables
   if (color.match(/var\(/)) {
     const regex = /var\(([^,]+),\s*([^)]+)\)/;
     const [, , colorValue] = color.match(regex);
-    return transparentColor(colorValue, opacity);
+    const value = colorValue as string;
+    const isValidColor = isHex(value) || isRgb(value);
+    return transparentColor(isValidColor ? value : `rgb(${value})`, opacity);
```

Comment:
I understand this code, but why is this necessary now?

Reply:
because it's broken without, it recieves "255, 255, 255" and need to parse this value